src/Symfony/Component/Console/Event/ConsoleExceptionEvent.php

```diff
@@ -27,12 +25,23 @@ class ConsoleExceptionEvent extends ConsoleEvent
     private $exception;
     private $exitCode;
 
-    public function __construct(Command $command, InputInterface $input, OutputInterface $output, \Exception $exception, $exitCode, $deprecation = true)
+    public function __construct($input, $output, $exception, $exitCode, $command = null, $deprecation = true)
     {
         if ($deprecation) {
             @trigger_error(sprintf('The %s class is deprecated since version 3.3 and will be removed in 4.0. Use the ConsoleErrorEvent instead.', __CLASS__), E_USER_DEPRECATED);
         }
 
+        if ($input instanceof Command) {
+            $tmpExitCode = $command;
+            $command = $input;
+            $input = $output;
+            $output = $exception;
+            $exception = $exitCode;
+            $exitCode = $tmpExitCode;
+
+            @trigger_error(sprintf('Passing an instance of %s as first argument of %s() is deprecated since version 3.3. Pass it as 5th argument instead.', Command::class, __METHOD__), E_USER_DEPRECATED);
```

Comment:
This deprecation is useless imo. People creating the class as `($command, $input, $output)` already get a deprecation, as the class shouldn't be initialized (ConsoleErrorEvent should be initialized instead).